packages/bundler/localconfig/bundler.rip7560.config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "gasFactor": "1",
+  "port": "3000",
+  "network": "http://127.0.0.1:8545",
+  "entryPoint": "0x0000000071727De22E5E9d8BAf0edAc6f37da032",
+  "beneficiary": "0xd21934eD8eAf27a67f0A70042Af50A1D6d195E81",
```

Comment:
change to default "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266" account (the test...junk)